src/views/product-integration/component-view/components/variable-group-list/style.module.css

```diff
@@ -90,3 +81,19 @@
 		}
 	}
 }
+
+.key {
+	composes: g-offset-scroll-margin-top from global;
```

Comment:
I had to un-nest `.key` styles, as `compose` can't be used with `.key`, and we need `g-offset-scroll-margin-top` for the anchor link to jump to the correct scroll position (accounting for the top nav with scroll-margin-top).